doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -423,7 +425,7 @@ Cellular samples (renamed from nRF9160 samples)
 
 * :ref:`nrf_cloud_rest_fota` sample:
 
-   * Added reporting of device and connection info to the device shadow.
+  * Added reporting of device and connection info to the device shadow.
```

Comment:
Repeating word 'module'
Line 432: 
 * A DTS overlay file for LEDs on the nRF9160 DK to be compatible with the :ref:`caf_leds` .